profile/isa_ro_crate_mapping.md

```diff
@@ -12,13 +12,14 @@ This is the mapping between ISA-json types defined in https://isa-specs.readthed
 |people|creator||
 |identifier|identifier||
 |description|description||
-|studies|hasPart||
+|studies|hasPart|MAY also directly point to assays|
```

Comment:
I think this could be confusing. Maybe we should say a bit more.

Reply:
Can't really think of anything to add that won't make it more confusing.